fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -42,6 +43,31 @@ public ConcretePhysicalTable(
     }
 
     /**
+     * Create a concrete physical table with provided availability.
+     *
+     * @param name  Name of the physical table as TableName
+     * @param timeGrain  time grain of the table
+     * @param columns  The columns for this table
+     * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
+     * @param availability the instance of Availability specifying the availabilities of table columns
+     */
+    protected ConcretePhysicalTable(
+            @NotNull TableName name,
+            @NotNull ZonedTimeGrain timeGrain,
+            @NotNull Set<Column> columns,
+            @NotNull Map<String, String> logicalToPhysicalColumnNames,
+            @NotNull Availability availability
+    ) {
+        super(
```

Comment:
Since we have this constructor now, can you change the `super` in the other constructor to `this`? Thanks.